geode-docs/tools_modules/http_session_mgmt/configuring_non_sticky_sessions.html.md.erb

```diff
@@ -19,7 +19,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-This topic describes the differences between stick and non-sticky session in Geode
+This section describes the configuration of non-stick sessions
```

Comment:
I'm confused about changing the title to "non-sticky" and changing the description to "non-stick."